movie-blog/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Common suggestion I make for styling components - to stick to CSS in JS approach (e.g. `styled-components`). You may expect the latter to perform and scale better and that may be a helpful exercise. You may safely ignore my recommendation, though.